crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -62,8 +63,10 @@ impl From<&StarknetContractArtifacts> for ContractArtifacts {
     }
 }
 
-pub struct CairoHintProcessor<'a> {
-    pub blockifier_syscall_handler: SyscallHintProcessor<'a>,
+// This hint processor provides an implementation logic for functions from std_forge library.
+// If cannot execute a hint it falls back to the CheatableSyscallHandler
```

Comment:
```suggestion
// It uses CheatableSyscallHandler to execute starknet related hints
```
At least that's how I understand it

Reply:
I am not sure if it will be true, I will keep in mind and update it later after I finish.